lib/events/event_router.rb

```diff
@@ -3,13 +3,14 @@ class EventRouter
     def self.dispatch(event_type, payload, headers = nil)
       case event_type
       when "Catalog.upload"
-        # TODO add service later
+        PersisterTaskService.new(payload).process
       when "Source.create"
         SourceCreateTaskService.new(payload).process
       when "Source.availability_check"
-        # TODO add service later
+        task = CheckAvailabilityTaskService.new(payload).process.task
```

Comment:
@hsong-rh Should we normalize this one by using payload[:params] so all of our services work with the same flat hash structure instead of an additional param node in between.